2025/docs/9-nhi-reuse.md

```diff
@@ -1 +1,61 @@
-TBA
\ No newline at end of file
+# NHI9:2025 NHI Reuse
+
+| Threat agents/Attack vectors | Security Weakness                                   | Impacts                                       |
+|------------------------------|-----------------------------------------------------|-----------------------------------------------|
+| Exploitability - **Hard**    | Prevalence - **Widespread** : Detectability - **Hard**  | Technical - **Low** : Business - **Specific** |
+| Successfully exploiting a Re-used NHI requires the threat agent to first gain access to the environment. Therefore, NHI Reuse is dependent on a separate initial access vector. |  NHIs are very commonly reused because tailor fitting NHI for each workload is difficult. Common cases include the use of a single AWS IAM Role for multiple workloads or the use of a single API key for multiple workloads. | NHI Reuse impact depends on the privilege of the associated NHI. If least-privilege is adopted, this impact is Low. |
```

Comment:
Please add the description for Detectability as a sentence after the description for Prevalence

Reply:
Thanks for spotting this mistake! It should be resolved in https://github.com/OWASP/www-project-non-human-identities-top-10/commit/84dee9a287c5fd0161bd2e350eeff8a966e19234